gst/elements/gvametapublish/pravega/gvametapublishpravega.cpp

```diff
@@ -43,10 +44,10 @@ class GvaMetaPublishPravegaPrivate {
 
     gboolean start() {
         GST_INFO_OBJECT(_base,
-                        "Gvametapushlish params: \n--controller-uri: %s\n --scope: %s\n --stream:%s \n --routing-key: %s\n",
-                        _controller_uri.c_str(), _scope.c_str(), _stream.c_str(), _routing_key.c_str());
+                        "Gvametapushlish params: \n--controller-uri: %s\n--keyclock_file: %s\n --scope: %s\n --stream:%s \n --routing-key: %s\n",
+                        _controller_uri.c_str(), _keyclock_file.c_str(), _scope.c_str(), _stream.c_str(), _routing_key.c_str());
 
-        _stream_manager = create_stream_manager(_controller_uri.c_str(), false, false, false);
+        _stream_manager = create_stream_manager(_controller_uri.c_str(), _keyclock_file.c_str(), true, true);
```

Comment:
do not hard code last two param as `true, true`

Reply:
Makes sense, explicitly ask user for all params now.